src/common/Icons.js

```diff
@@ -8,6 +8,7 @@ import type { FeatherGlyphs } from 'react-native-vector-icons/Feather';
 import IoniconsIcon from 'react-native-vector-icons/Ionicons';
 import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
 import SimpleLineIcons from 'react-native-vector-icons/SimpleLineIcons';
+import FontAwesome5 from 'react-native-vector-icons/FontAwesome5';
```

Comment:
```
avatars [nfc]: Add muted user placeholder avatars.
```

nit: this commit isn't NFC 🙂

Reply:
I got an error with "Unrecognized font family" on iOS, so I followed a [troubleshooting entry](https://github.com/oblador/react-native-vector-icons#troubleshooting) and saw that we hadn't added some things to our Info.plist. This was sufficient to get it working:

```diff
--- ios/ZulipMobile/Info.plist
+++ ios/ZulipMobile/Info.plist
@@ -63,6 +63,9 @@
                <string>Entypo.ttf</string>
                <string>EvilIcons.ttf</string>
                <string>FontAwesome.ttf</string>
+               <string>FontAwesome5_Brands.ttf</string>
+               <string>FontAwesome5_Regular.ttf</string>
+               <string>FontAwesome5_Solid.ttf</string>
                <string>Foundation.ttf</string>
                <string>Ionicons.ttf</string>
                <string>Feather.ttf</string>
```

Reply:
Shouldn't be needed now that I'm not using FontAwesome5 (see [CZO thread](https://chat.zulip.org/#narrow/stream/101-design/topic/Hiding.20user-muted.20messages))